core/src/main/kotlin/org/evomaster/core/output/clustering/metrics/DistanceMetricLastLine.kt

```
@@ -13,9 +13,12 @@ import org.evomaster.core.problem.rest.RestCallResult
  *
  */
 
-class DistanceMetricLastLine : DistanceMetric<RestCallResult>() {
+class DistanceMetricLastLine(
+        epsilon: Double = 0.8
+) : DistanceMetric<RestCallResult>() {
     private val name = "LastLine"
-    private var recommendedEpsilon = 0.8
+    private var recommendedEpsilon = if(epsilon in 0.0..1.0)  epsilon
+                                        else 0.8
```

same as previous... rather throw an exception